packages/arb-token-bridge-ui/src/components/TransferPanel/ETHPrice.tsx

```diff
@@ -0,0 +1,43 @@
+import { useMemo } from 'react'
+
+import { useNativeCurrency } from '../../hooks/useNativeCurrency'
+import { useNetworks } from '../../hooks/useNetworks'
+import { useNetworksRelationship } from '../../hooks/useNetworksRelationship'
+import { useAppState } from '../../state'
+import { isNetwork } from '../../util/networks'
+import { useETHPrice } from '../../hooks/useETHPrice'
+import { formatUSD } from '../../util/NumberUtils'
+
+export function ETHPrice({
```

Comment:
Can it be a hook instead? Seems like this component is mostly about the logic, and doesn't do any complicated UI